main.tf

```
@@ -281,6 +290,15 @@ resource "azurerm_windows_web_app" "this" {
         }
       }
     }
+
+    dynamic "cors" {
+      for_each = var.cors
+
+      content {
+        allowed_origins     = cors.value.allowed_origins
+        support_credentials = cors.value.support_credentials
+      }
+    }
```

Could we split `cors` variable into two simpler variables `cors_allowed_origins` and `cors_support_credentials`, then add some logic to create the block when needed?

```suggestion
    dynamic "cors" {
      for_each = length(var.cors_allowed_origins) > 0 || var.cors_support_credentials ? [0] : []

      content {
        allowed_origins     = var.cors_allowed_origins
        support_credentials = var.cors_support_credentials
      }
    }
```

Or could we even hardcode the `cors` block?